src/components/studyDetails/DataSet.tsx

```diff
@@ -113,37 +116,9 @@ const DataSetComponent: React.FC<DatasetComponentProps> = ({
         }
         return '';
     };
-    /*
-    const addDatasetToStudy = (row: any) => {
-        setUpdateCache({ ...updateCache, [getStudyByIdUrl(study.id)]: true, [getDatasetsInStudyUrl(study.id)]: true });
-        setIsOpen(false);
-        if (row && !checkIfDatasetIsAlreadyAdded(row.id)) {
-            const studyId = window.location.pathname.split('/')[2];
-            const datasetList: any = [...study.datasets];
-            datasetList.push(row);
-            setStudy({ ...study, datasets: datasetList });
-            addStudyDataset(studyId, row.id).then((result: any) => {
-                if (result && result.message) {
-                    console.log('Err');
-                }
-                datasetsResponse.setLoading(false);
-            });
-        }
-    };
-    const checkIfDatasetIsAlreadyAdded = (id: string) => {
-        let elementExist = false;
-        study.datasets &&
-            study.datasets.forEach((element: any) => {
-                if (element.id === id) {
-                    elementExist = true;
-                }
-            });
-        return elementExist;
-    };
-*/
```

Comment:
This is a feature that is hidden in this MVP, please put it back!

Reply:
Sure, I will put it back. I suggest we use feature toggles in the future. Sometimes commented code gives the impression that we do not need the block of code.

Reply:
Do you have a link to something you have used/that we can use?

Reply:
Something like this, 

https://medium.com/ecovadis-engineering/using-feature-toggle-in-a-react-application-part-1-ee34a0e72cf4